code/index.html

```diff
@@ -2,11 +2,163 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
-  </body>
-</html>
+    <title>Chefinne - P3</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">     
+<header>
+<div class="hamburgerbackground">
+    <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </div>
+</div>
+    <div class="navigation">
+        <menu>
+            <a href="./Home.html">Home</a>
+            <a href="./Uber Uns.html">Über Uns</a>
+            <a href="./Events.html">Events</a>
+            <a href="./News.html">News</a>
+            <a href="./Inspiration & Empfehlungen.html">Inspiration & Empfehlungen</a>
+            
+                <div class="search">
+                    <input class="search-field" type="text" placeholder="Search... " name="search">
+                    <button class="search-button" type="button">🔍</button>
+                </div>  
+        
+        </menu>
+        
+    </div>
+</header>
+</head>
+<body> 
+    <section class="intro-container">
+        <div>
+            <h2>A girl gang of women agressively supporting eachother</h2>
+            <h3>In jeder von uns steckt eine Chefin</h3>
+            <p>Netzwerk von jungen, dynamischen Frauen, die sich gegenseitig unterstützen und stärken, nie ausgelernt haben und sich gerne auch mal selber feiern.</p>
+        </div>
+        <!--Kommentar für meine Übersicht oder Struktur-->
+        
+        <div>
+            <p>
+            Auf <a href="https://www.linkedin.com/groups/8967842/" target="_blank" rel="noopener noreferrer">LinkedIn</a> und WhatsApp teilen wir alles, was uns bewegt.<br>
+            <br>Wir freuen uns von dir zu hören! 
+            </p>
+    </section>
+
+    <section class="second-container">
+        <div class="second-container-left">
+            <h1>Weihnachtsfeier 2019</h1>
+            <img src="Images/Bild Weihnachtsfeier.jpg" alt="Bild von Chefinnen-Members an Weihnachtsfeier">
```

Comment:
- We should not have spaces in our file names
- We need to specify the relative path to our assets. Since this is located in a sibling folder to the code folder, the path is `"../Images/Bild von Chefinnen-Members an Weihnachtsfeier"`

Same goes for all your images :)